python/lsst/ap/association/diaCalculationPlugins.py

```diff
@@ -58,6 +60,24 @@
            "SigmaDiaTotFlux", "SigmaDiaTotFluxConfig")
 
 
+def catchNumpyWarnings(_func=None, *, warnings=[]):
+    """Decorator for generically catching numpy warnings.
+    """
+    def decoratorCatchWarnings(func):
+        @functools.wraps(func)
+        def wrapperCatchWarnings(*args, **kwargs):
+            np.warnings.catch_warnings()
```

Comment:
Make this `with np.warnings.catch_warnings():` and indent the `for` loop and `return` statement below.

Reply:
Thanks for catching this. Definitely a bug I introduced while re-writing this.